package.json

```diff
@@ -71,26 +74,24 @@
     "@vitest/browser": "^0.34.6",
     "@vitest/coverage-istanbul": "^0.34.6",
     "@vitest/ui": "^0.34.6",
+    "@xml-tools/parser": "^1.0.11",
+    "chevrotain": "7.1.1",
```

Comment:
I'm not for sure to understand why this package is required on development and the version is locked here.

Reply:
Can you give an example of what the parser doesn't support? I'm not sure I understand.

Reply:
> what the parser doesn't support?

1. Blank line before prologs: https://github.com/SAP/xml-tools/issues/248
2. Prologs only: 
    ```ts
    /* xml */ `<?xml version="1.0" encoding="utf-8" ?>`
    ```
3. Multiple (root) elements (fragments): 
    ```ts
    /* xml */ `
    <component-a></component-a>
    <component-b></component-b>
    `
    ```

Reply:
Ahh I see.

If you want to upstream the ability to handle these errors in the prettier XML plugin, I would accept contributions for them.

Reply:
But I think we can discuss this in another issue or discussion? Because this PR is about adding support for new languages like `toml` and `ini`. I just thought it would be appropriate to mention your guys to let you know this work is based on your plugins. Apologize if this bothers you. What do you think?